gorp/client.go

```diff
@@ -218,7 +242,7 @@ func (c *Client) SaveLogMultipart(log []*SaveLogRQ, files map[string]*os.File) (
 		if _, sErr := os.Stat(v.Name()); os.IsNotExist(sErr) {
 			return nil, fmt.Errorf("file %s does not exist", v.Name())
 		}
-		rq.SetMultipartField(k, k, "", v)
+		rq.SetMultipartField("file", k, mime.TypeByExtension(filepath.Ext(k)), v)
```

Comment:
RP accepts multiple files, so let's keep part name dynamic (key in the map provided)